inst/rmd/h/default.html

```diff
@@ -110,6 +112,7 @@
 
 $if(theme)$
 <style type="text/css">
+$if(bs3)$
 h1 {
   font-size: 34px;
 }
```

Comment:
Even if bs4, make sure h1.title is bigger than h1

Reply:
Done in https://github.com/rstudio/rmarkdown/pull/1706/commits/87f8b0f1ff294e07fcdbcebc64ba333c9393390e